subtrack.MAUI/wwwroot/css/app.css

```
@@ -38,6 +38,10 @@ a, .btn-link {
     color: red;
 }
 
+.detail-label-fs {
+    font-size: 0.875em;
```

```suggestion
    font-size: 0.88em;
```

Same as before, apply in order to ensure i stay sane 😂